web/templates/stream_settings/stream_settings.hbs

```diff
@@ -39,7 +39,7 @@
                   invite_only=invite_only
                   is_web_public=is_web_public }}
                 <div class="stream-name">
-                    <span class="sub-stream-name" title="{{name}}">{{name}}</span>
+                    <span class="sub-stream-name">{{name}}</span>
```

Comment:
Do not remove the `name` tooltip but instead show it with EXTRA_LONG_HOVER_DELAY since sometimes we can have very long stream names. See https://github.com/zulip/zulip/pull/27913#issuecomment-1829552191.

Reply:
@shashank-23002 I think you have missed this feedback.